CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
 
+## 9.0.1 - March 6, 2020
+[Changes](https://github.com/mapbox/mapbox-gl-native-android/compare/android-v9.0.0...android-v9.0.1) since [Mapbox Maps SDK for Android v9.0.0](https://github.com/mapbox/mapbox-gl-native-android/releases/tag/android-v9.0.0):
+### Bug fixes
+ - Fix Proguard configuration to prevent obfuscation of telemetry event classes. [#195](https://github.com/mapbox/mapbox-gl-native-android/pull/195)
+### Other changes
+ - Update mapbox-gl-native to v1.2.2. [#216](https://github.com/mapbox/mapbox-gl-native-android/pull/216)
```

Comment:
Instead of this entry, we should take the bug fixes in the changelog of GL-Native v1.2.2 and add it to our changelog. 

So delete this entry and section, and under "Bug fixes", add:
```
- Fix a crash that could occur when using styles with `LinePattern`. [#16249](https://github.com/mapbox/mapbox-gl-native/pull/16249)
```